packages/p2p/src/components/order-details/order-details.jsx

```diff
@@ -292,7 +292,7 @@ const OrderDetails = observer(() => {
                                     label={labels.right_send_or_receive}
                                     value={`${amount_display} ${account_currency}`}
                                 />
-                                <OrderInfoBlock label={localize('Time')} value={purchase_time} />
```

Comment:
could we keep purchase_time instead of order_purchase_time? cause its gonna cause conflicts with the other card 🙏